themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md

```diff
@@ -0,0 +1,17 @@
+The `chef-backend.rb` file is generated using
+`chef-backend-ctl gen-sample-backend-config` and controls most of the
+various feature and configuration flags going into a Chef HA backend
+node. A number of these options control the reliability, stability and
+uptime of the backend PostgreSQL databases, the Elasticsearch index,
+and the leader election system. Please refrain from changing them unless
+you have been advised to do so.
+
+
+-   `fqdn` Host name of this node.
+-   `hide_sensitive` Set to `false` if you wish to print deltas of
+    sensitive files and templates during `chef-backend-ctl reconfigure`
+    runs. `true` by default.
+-   `ip_version` Set to either `'ipv4'` or `'ipv6'`. `'ipv4'` by
+    default.
+-   `publish_address` Externally resolvable IP address of this back-end
+    node.
```

Comment:
Are these settings that the user could or should change?
I'm thinking prefacing this list with something like:
`Verify that the following settings are correctly configured:`
or
`Configure the following settings:`

Reply:
I think it is more like "these are the only settings you should change when setting up without guidance" - @sean-horn ?